views-thymeleaf/src/main/java/io/micronaut/views/thymeleaf/ThymeleafViewsRenderer.java

```diff
@@ -52,20 +53,24 @@
 
     protected final AbstractConfigurableTemplateResolver templateResolver;
     protected final TemplateEngine engine;
+    protected final HttpLocaleResolver httpLocaleResolver;
     protected ResourceLoader resourceLoader;
 
     /**
      * @param templateResolver   The template resolver
      * @param templateEngine     The template engine
      * @param resourceLoader     The resource loader
+     * @param httpLocaleResolver The locale resolver
      */
     @Inject
     public ThymeleafViewsRenderer(AbstractConfigurableTemplateResolver templateResolver,
```

Comment:
This is a breaking change you need to leave the old constructor and flag it as `@Deprecated` and create a new constructor and annotate the new one with `@Inject`. Leave the old one without `@Inject`annotation.

Reply:
Fixed in the most recent push. In order to remain consistent with the previous logic, the deprecated constructor passes in an HttpLocaleResolver which only looks at the request's locale and defaults to US.

Reply:
thanks for doing the change.